Tests/XCUITests/HistoryTests.swift

```diff
@@ -103,215 +98,214 @@ class HistoryTests: BaseTestCase {
         XCTAssertEqual(app.alerts.buttons["OK"].exists, false)
     }
 
-    func testRecentlyClosedOptionAvailable() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
-        navigator.goto(HistoryRecentlyClosed)
-        waitForNoExistence(app.tables["Recently Closed Tabs List"])
+    func testRecentlyClosedWebsiteOpen() {
+        openBookOfMozilla()
 
-        // Go to the default web site  and check whether the option is enabled
-        navigator.nowAt(LibraryPanel_History)
-        navigator.goto(HomePanelsScreen)
-        navigator.openURL(path(forTestPage: "test-mozilla-book.html"))
-        navigator.nowAt(BrowserTab)
-        navigator.goto(BrowserTabMenu)
-        // Workaround to bug 1508368
-        navigator.goto(LibraryPanel_Bookmarks)
-        navigator.goto(HomePanelsScreen)
-        navigator.performAction(Action.OpenNewTabFromTabTray)
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
+        // The tab, which is still opened, is not included in the "Recently Closed Tabs List"
+        navigator.nowAt(HomePanelsScreen)
         navigator.goto(LibraryPanel_History)
-        navigator.goto(HistoryRecentlyClosed)
-        waitForNoExistence(app.tables["Recently Closed Tabs List"])
-        navigator.nowAt(LibraryPanel_History)
-        navigator.goto(HomePanelsScreen)
+        waitForExistence(app.tables[HistoryPanelA11y.tableView])
+        XCTAssertFalse(app.tables.cells.staticTexts[bookOfMozilla["label"]!].exists)
+        XCTAssertTrue(app.tables[HistoryPanelA11y.tableView].staticTexts[emptyRecentlyClosedMesg].exists)
+    }
 
-        // Now go back to default website close it and check whether the option is enabled
-        navigator.openURL(path(forTestPage: "test-mozilla-book.html"))
-        waitUntilPageLoad()
-        waitForTabsButton()
-        navigator.goto(TabTray)
-        navigator.performAction(Action.AcceptRemovingAllTabs)
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
-        navigator.goto(BrowserTabMenu)
+    func testRecentlyClosedWebsiteClosed() {
+        openBookOfMozilla()
+
+        closeFirstTabByX()
+
+        // On regular mode, the closed tab is listed in "Recently Closed Tabs List"
+        navigator.nowAt(HomePanelsScreen)
         navigator.goto(HistoryRecentlyClosed)
+        waitForExistence(app.tables["Recently Closed Tabs List"])
+        XCTAssertTrue(app.tables.cells.staticTexts[bookOfMozilla["label"]!].exists)
+    }
 
-        // The Closed Tabs list should contain the info of the website just closed
-        waitForExistence(app.tables["Recently Closed Tabs List"], timeout: 3)
-        XCTAssertTrue(app.tables.cells.staticTexts[closedWebPageLabel].exists)
+    func testRecentlyClosedPrivateMode() {
```

Comment:
This test is not correctly design. The flow would be:
- open tab in regular mode, close it
- open tab in private mode, close it
- go to recently closed menu
- only the regular tab should appear in that menu

You can simplify and just test that if you open a tab in private mode and close it, it is not shown in the menu. But the tab has to be open in private mode, so you have to go to that menu first and open the tab from there or use the action that we have         `navigator.performAction(Action.OpenPrivateTabLongPressTabsButton)`  from Homescreen

Reply:
Just confirming if this is the simplified case.

https://user-images.githubusercontent.com/1740517/197610780-1bfe6b58-caaf-4ea3-82f1-0b55b8a47287.mov

Reply:
Yes, I would say that would be the correct user flow, since we are testing on other tests the first bullet point I mentioned about closing a regular tab.